src/components/common/DatePicker.vue

```diff
@@ -30,7 +30,7 @@
       <v-date-picker width="490" v-model="dateText">
         <template v-slot:default>
           <div>
-            <v-btn text color="primary" @click="emitDate(dateText)"><strong>Ok</strong></v-btn>
+            <v-btn text color="primary" @click="emitDate(dateText)"><strong>OK</strong></v-btn>
```

Comment:
Aw man! I thought Ok was cute ;)

Reply:
Yea, i'm disappointed with this change too... 
Gotta choose our battles..